client/src/app/site/motions/components/motion-poll/motion-poll.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { MotionPollComponent } from './motion-poll.component';
+import { E2EImportsModule } from 'e2e-imports.module';
+import { MetaTextBlockComponent } from '../meta-text-block/meta-text-block.component';
+
+describe('MotionPollComponent', () => {
+    // TODO testing fails if personalNotesModule (also having the MetaTextBlockComponent)
```

Comment:
Here I could use some help in angular magic

Reply:
After discussion I will comment this test for now, and we should collect an issue with missing/failing tests due to async/angular magic